src/diagnostics/riscv/disasm-riscv.cc

```diff
@@ -129,6 +129,10 @@ class Decoder {
   void DecodeCSType(Instruction* instr);
   void DecodeCJType(Instruction* instr);
   void DecodeCBType(Instruction* instr);
+  // (B)itmanip extension
+  bool DecodeBRType();
```

Comment:
Decoders here might probably have `Instruction* instr` as an input param in contrary to `simulator`.

See also implementations `bool Decoder::DecodeBRType()` and `bool Decoder::DecodeBIType()` below